src/transactionEvents.ts

```diff
@@ -0,0 +1,41 @@
+import { Address } from "./address";
+
+export class TransactionEvent {
+    address: Address = Address.empty();
+    identifier: string = "";
+    topics: Uint8Array[] = [];
+
+    data: Uint8Array = new Uint8Array();
+    additionalData: Uint8Array[] = [];
+
+    constructor(init?: Partial<TransactionEvent>) {
+        Object.assign(this, init);
+    }
+
+    static fromHttpResponse(responsePart: {
```

Comment:
In sdk-py, this conversion is defined somewhere else (network providers module).